uaclient/entitlements/base.py

```diff
@@ -132,7 +132,7 @@ def incompatible_services(self) -> Tuple[IncompatibleService, ...]:
         return self._incompatible_services
 
     @property
-    def required_services(self) -> Tuple[str, ...]:
+    def required_services(self) -> Tuple[Type["UAEntitlement"], ...]:
```

Comment:
Might be a nit-pick, is it possible to have no speech marks on the type to be consistent with line 62?

Reply:
The reason for that is that we are using a "type" of the same class that is being created. That's why we need to delay the type registration using ```"UAEntitlement"```